Gemfile

```diff
@@ -4,12 +4,16 @@ group :test do
   gem "json_pure", '2.0.1' if RUBY_VERSION < '2.0'
   gem "rake"
   gem "puppet", ENV['PUPPET_VERSION'] || '~> 3.8.4'
+  gem 'hiera'
```

Comment:
Why do you add hiera? It's already a dependency of the puppet gem. It seems pointless to me.